internal/venice-common/src/main/java/com/linkedin/venice/blob/ServerBlobFinder.java

```diff
@@ -26,29 +23,21 @@ public BlobPeersDiscoveryResponse discoverBlobPeers(String storeName, int versio
     BlobPeersDiscoveryResponse response = new BlobPeersDiscoveryResponse();
     try {
       String currentVersionResource = Version.composeKafkaTopic(storeName, version);
-      // Get the partition assignments for the current version
```

Comment:
It looks like there are still a lot of changes here unrelated to this PR, did you rebase it or it is intended?

Especially for this one, as a simple POJO class "BlobPeersDiscoveryResponse", ideally we should only have setter/getter class in that class, we should not plant other complex logic there. Could you share if there is any particular reason to do so? If no, I'd suggest let's revert it back and move the addPartition() logic back into the ServerBlobFinder implementation where it should have such processing logic.

Reply:
Changed logic back